components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -3273,6 +3274,8 @@ ledger::ClientInfoPtr GetDesktopClientInfo() {
     info->os = ledger::OperatingSystem::UNDEFINED;
   #endif
 
+  info->channel = brave::GetChannelName();
```

Comment:
`chrome::GetChannelName` should be passed to RewardsServiceImpl in the factory to eliminate this layering violation

Reply:
@bridiver raised https://github.com/brave/brave-browser/issues/12769